tests/testthat/test-shinytest2-tm_a_pca.R

```diff
@@ -0,0 +1,171 @@
+testthat::test_that("e2e - tm_a_pca: data extract changes output", {
```

Comment:
I think this test does more then what is indicated here. Would be nice to split display-changing encoding inputs to their own testthat scope. WDYT?

Reply:
I don't think it does, all of the data extracts are in the same test. 

It might broaden the scope... Ideally, this test would be split into 3, but the e2e is very slow and I'm worried about the overall impact on development on {tmg}

---

On a different note, it might be better to use snapshots of html/images instead of only testing for no errors.

WDYT?

Reply:
Tests were split in more with an added helper `local_app_driver_*` functions to reuse for each module.